minerl/Malmo/Minecraft/src/main/java/com/microsoft/Malmo/Server/ServerStateMachine.java

```diff
@@ -809,6 +809,7 @@ private EntityPlayerMP getPlayerFromUsername(String username)
 
         private void initialisePlayer(String username, String agentname)
         {
+            System.out.println("Initializing player with username " + username + " and agentname " + agentname);
```

Comment:
We should ideally log these things instead of relying on std out but since this is captured by the logger currently we can refactor all of the printing later

Reply:
https://github.com/openai/openai/projects/2#card-48563180